hugegraph-api/src/main/java/com/baidu/hugegraph/api/filter/AuthenticationFilter.java

```diff
@@ -183,12 +193,14 @@ public Principal getUserPrincipal() {
 
         @Override
         public boolean isUserInRole(String required) {
+            /*
             if (required.equals(HugeAuthenticator.KEY_DYNAMIC)) {
                 // Let the resource itself determine dynamically
                 return true;
             } else {
                 return this.matchPermission(required);
-            }
+            }*/
```

Comment:
delete if not used.
If want to keep origin, please add comments to describe the reason